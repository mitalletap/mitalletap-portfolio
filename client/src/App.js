import React, { Component } from 'react';
import { BrowserRouter as Router, Switch, Route, Link, withRouter } from 'react-router-dom';
import logo from './logo.svg';
import './App.css';
import Home from './components/Home';
import Admin from './components/Admin';

class App extends Component {

  render() { 
    return (  
      <div className="app-container">
      <Router>
        <Link to="/"></Link>
        <Link to="/admin"></Link>
        <Switch className="app-content-items">
          <Route exact path="/"><Home /></Route>
          <Route exact path="/admin"><Admin /></Route>
        </Switch>
      </Router>
      </div>
    );
  }
}
 
export default App;