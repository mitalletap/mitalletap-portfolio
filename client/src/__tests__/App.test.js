import React, { Component } from 'react';
import { shallow, mount } from 'enzyme';
import App from '../App';

describe('tests the app page', () => {

    let app;
    let mockData;
    let instance;
    
    beforeEach(() => {
        app = mount(<App />);
    });

    afterEach(() => {
        
    });

    it('should match snapshot', () => {
        expect(app).toMatchSnapshot();
    });

});