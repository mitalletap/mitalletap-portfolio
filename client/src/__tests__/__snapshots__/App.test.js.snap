// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests the app page should match snapshot 1`] = `ReactWrapper {}`;
